web/src/components/SkipToMainContent.test.tsx

```diff
@@ -0,0 +1,28 @@
+import { SkipToMainContent } from "@/components/SkipToMainContent";
+import analytics from "@/lib/utils/analytics";
+import { fireEvent, render, screen } from "@testing-library/react";
+
+jest.mock("@/lib/utils/analytics", () => setupMockAnalytics());
+
+const mockAnalytics = analytics as jest.Mocked<typeof analytics>;
+function setupMockAnalytics(): typeof analytics {
+  return {
+    ...jest.requireActual("@/lib/utils/analytics").default,
+    event: {
+      ...jest.requireActual("@/lib/utils/analytics").default.event,
+      skip_to_main_content: {
+        click: {
+          skip_to_main_content: jest.fn(),
+        },
+      },
+    },
+  };
+}
+
+describe("<SkipToMainContent>", () => {
+  it("fires skip_to_main_content analytics event when link is clicked", () => {
+    render(<SkipToMainContent />);
+    fireEvent.click(screen.getByText("Skip to main content"));
```

Comment:
Shouldn't this grab the string from Config? This will break if the content team changes the name right?

Reply:
That is 100% why I added the string to config in the first place and then promptly didn't add it to the test. Thank you!